__tests__/app.test.js

```diff
@@ -144,5 +164,25 @@ describe("NC-News", () => {
           expect(body.msg).toBe("Bad Request");
         });
     });
+    test("status 404 POST/api/articles/99999/comments responds with an error when that article Id does not exist", () => {
+      const newComment = { username: "rogersop", body: "Hello" };
+      return request(app)
+        .post("/api/articles/99999/comments")
+        .send(newComment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Article not found");
+        });
+    });
+    test("status 400 POST/api/articles/1/comments responds with an error when is a malformed body or the user is missing required fields", () => {
+      const newComment = {};
+      return request(app)
+        .post(`/api/articles/1/comments`)
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad Request");
+        });
+    });
```

Comment:
it would be nice to have a couple of tests here for testing if the username was included in the object to send but not valid and a bad path test too